py/server/deephaven/experimental/s3.py

```diff
@@ -84,6 +89,28 @@ def __init__(self,
             num_concurrent_write_parts (int): the maximum number of parts that can be uploaded concurrently when writing
                 to S3 without blocking, defaults to 64. Setting a higher value may increase throughput, but may also
                 increase memory usage.
+            profile_name (str): the default profile name used for configuring the default region, credentials, etc.,
+                when reading or writing to S3. If not provided, the AWS SDK picks the profile name from the
+                'aws.profile' system property, the "AWS_PROFILE" environment variable, or defaults to "default".
```

Comment:
what does "default" mean?

Reply:
There are at least two defaults that are ambiguous.
1. "default profile name".  Should this just be "profile name"?  If not, what makes it a default?
2. "defaults to \"default\"".  What does the last default mean?